### PR TITLE
Package varlink

### PR DESCRIPTION
**Pull request by h-vetinari**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by h-vetinari** [comment]
@haraldh @4383
If you want to share maintainership of this recipe for varlink I'd be more than happy to share that role. :)

**Comment by 4383** [comment]
@h-vetinari WFM

**Comment by h-vetinari** [comment]
@4383 
I'm interpreting "WFM" as "works for me". If this is not accurate, please let me know.

@conda-forge-admin, please rerender

**Comment by h-vetinari** [comment]
@conda-forge/help-python 

Not sure what this recipe failure is about. Could anyone take a look (also at this staged recipe in general)...?

File: recipes/varlink/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  license_family: "APACHE"
+  license_file: "LICENSE"
+  summary: "Varlink"
+  doc_url: "https://varlink.github.io/python/"
+  dev_url: "https://github.com/varlink/python/"
+
+extra:
+  recipe-maintainers:
+    - h-vetinari
+    - 4383
```
Comment (isuruf) [inline recipes/varlink/meta.yaml]: @h-vetinari, I think conda-smithy doesn't like an integer here. Can you open an issue in https://github.com/conda-forge/conda-smithy ?

**Comment by h-vetinari** [comment]
@isuruf 
Thanks for the quick response, I've opened conda-forge/conda-smithy#1099.

File: recipes/varlink/meta.yaml (hunk 1 above)
Comment (isuruf) [reply recipes/varlink/meta.yaml]: Btw, try quoting here. `- "4383"`

**Comment by 4383** [comment]
> @4383
> I'm interpreting "WFM" as "works for me". If this is not accurate, please let me know.
@h-vetinari you are right

**Comment by haraldh** [comment]
> @haraldh @4383
> If you want to share maintainership of this recipe for varlink I'd be more than happy to share that role. :)

Don't have much time for maintaining it here, but can help, if something is not working as expected in varlink.

**Comment by h-vetinari** [comment]
@haraldh
Thanks for the response. I'd ask you the same thing as @4383 above:

> ... if there's more essential tests for varlink that you'd recommend (maybe not exactly the whole test suite, but important imports/linkages etc.), please let us know here.

**Comment by scopatz** [comment]
Thanks all!

**Pull request merged**